crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -79,7 +79,7 @@ def version
 
       def addons
         [].tap do |list|
-          ["ceph", "ha"].each do |addon|
+          ["ha"].each do |addon|
```

Comment:
Does it make sense to keep this instead of just doing it with `ha` directly? I dont think we expect more addons in the future.

Reply:
I thought about this, and decided to rather keep it to have it similar with 4.0 code